### PR TITLE
Added delete-group and create-group commands

### PR DESCRIPTION
**Pull request by PM-IST-PC**
## Status
Ready

## Related Issues
N/A

## Description
Adds two commands (Create Group, Delete Group)

## Screenshots
N/A

## Related PRs
N/A

## Required version of Demisto
Current (4.5)+

## Does it break backward compatibility?
No

## Must have
N/A

## Dependencies
N/A

## Additional changes
N/A

## Technical writer review
N/A

### CODE REVIEW

File: Integrations/Active_Directory_Query/Active_Directory_Query.yml [hunk 1]
```diff
@@ -393,5 +393,37 @@ script:
       specified group
   runonce: false
```
Comment (yaakovi) [inline Integrations/Active_Directory_Query/Active_Directory_Query.yml]: remove lines 394-395. they also appear on lines 426-427.

**Comment by yaakovi** [comment]
nice job

File: Integrations/Active_Directory_Query/Active_Directory_Query.yml [hunk 2]
```diff
@@ -391,7 +391,41 @@ script:
       type: string
     description: Retrieves the list of users or computers that are members of the
       specified group
-  runonce: false
+ - name: ad-create-group
```
Comment (yaakovi) [inline Integrations/Active_Directory_Query/Active_Directory_Query.yml]:
```suggestion
   - name: ad-create-group
```

Comment (yaakovi) [reply Integrations/Active_Directory_Query/Active_Directory_Query.yml]: this should solve your build problem.

**Comment by yaakovi** [comment]
@PM-IST-PC nice work, @kirbles19 is our technical writer and will go over your changes as well.

@kirbles19 please review the new command sections in the yml and the change log

**Comment by richardbluestone** [comment]
Hi, in the **ad-create group** **dn** argument, it is defined as "The Group's Full Distinguished Name (DN); Use " instead of ' when initializing this". What do you mean Use " instead of ' when initializing this"?. Can you give an example?

**Comment by PM-IST-PC** [comment]
The command will fail to run if you wrap the DN in single quotes rather than full quotes. Found during testing and added by CSE.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by richardbluestone** [comment]
The yml file had been reviewed and updated.

File: Integrations/Active_Directory_Query/Active_Directory_Query.yml [hunk 3]
```diff
@@ -101,329 +100,325 @@ script:
...
         the username argument must be specified.
     - name: group-cn
       required: true
-      description: The name of the group to add the user to
+      description: The name of the group for which to add the user.
     - name: base-dn
-      description: Root (e.g., DC=domain,DC=com). By default, the Base DN configured for the instance
-        will be used.
+      description: Root. For example, DC=domain,DC=com. By default, the Base DN configured for the instance
+      is used.
```
Comment (yaakovi) [inline Integrations/Active_Directory_Query/Active_Directory_Query.yml]:
```suggestion
        is used.
```

File: Integrations/Active_Directory_Query/Active_Directory_Query.yml [hunk 4]
```diff
@@ -101,297 +100,327 @@ script:
...
+      description: The name of the attribute to modify. For example, sn, displayName, mail, and so on.
     - name: attribute-value
       required: true
-      description: "The value the attribute should be changed to\t"
+      description: "The value of the attribute for which to change."
     - name: base-dn
-      description: Root (e.g. DC=domain,DC=com). By default, the Base DN configured for the instance
-        will be used.
+      description: Root. For example, DC=domain,DC=com. By default, the Base DN configured for the instance
+       is used.
```
Comment (yaakovi) [inline Integrations/Active_Directory_Query/Active_Directory_Query.yml]:
```suggestion
        is used.
```

File: Integrations/Active_Directory_Query/Active_Directory_Query.yml [hunk 5]
```diff
@@ -101,297 +100,327 @@ script:
...
   - name: ad-disable-account
     arguments:
     - name: username
       required: true
-      description: "The username of the account to disable (sAMAccountName)\t"
+      description: "The username of the account to disable (sAMAccountName)."
     - name: base-dn
       description: Root (e.g., DC=domain,DC=com). By default, the Base DN configured for the instance
-        will be used.
+      is used.
```
Comment (yaakovi) [inline Integrations/Active_Directory_Query/Active_Directory_Query.yml]:
```suggestion
        is used.
```

**Comment by yaakovi** [comment]
@PM-IST-PC please look at the circleci-build and fix lint errors

File: Integrations/Active_Directory_Query/Active_Directory_Query.py [hunk 6]
```diff
@@ -612,6 +612,42 @@ def create_contact():
...
 
 
+def create_group():
+    assert conn is not None
+    args = demisto.args()
+
+    object_classes = ["top", "group"]
+    dn = args.get('dn')
+    group_name = args.get('name')
+    group_class = args.get("group-class")
```
Comment (yaakovi) [inline Integrations/Active_Directory_Query/Active_Directory_Query.py]: @PM-IST-PC 
what is the purpose of this argument? it is not being used.

please update the code or remove the argument.

**Pull request merged**